src/fetchMOTD.ts

```diff
@@ -51,7 +51,7 @@ export function fetchMOTD(): void {
         configstore.set("motd.fetched", Date.now());
       })
       .catch((err) => {
-        utils.logWarning("Unable to fetch the CLI MOTD and remote config.");
+        utils.logWarning("Unable to fetch the CLI MOTD and remote config. This is not a fatal error, but may indicate an issue with your proxy set up.");
```

Comment:
It's not always a proxy. But "network connection" may be generic enough.

```suggestion
        utils.logWarning("Unable to fetch the CLI MOTD and remote config. This is not a fatal error, but may indicate an issue with your network connection.");
```

Also, fix the formatting errors.